components/selection/selection-observer-mixin.js

```diff
@@ -21,13 +26,18 @@ export const SelectionObserverMixin = superclass => class extends superclass {
 		super.connectedCallback();
 		// delay subscription otherwise import/upgrade order can cause selection mixin to miss event
 		requestAnimationFrame(() => {
-			const evt = new CustomEvent('d2l-selection-observer-subscribe', {
-				bubbles: true,
-				composed: true,
-				detail: {}
-			});
-			this.dispatchEvent(evt);
-			this._provider = evt.detail.provider;
+			if (!this.subscribedTo) {
+				const evt = new CustomEvent('d2l-selection-observer-subscribe', {
+					bubbles: true,
+					composed: true,
+					detail: {}
+				});
+				this.dispatchEvent(evt);
+				this._provider = evt.detail.provider;
+			} else {
```

Comment:
I think it'd be better to move this part into the `updated()` callback so that if the `subscribed-to` value changes you can unsubscribe and re-subscribe to the new provider. It's an edge case though.